lib/internal/process/promises.js

```diff
@@ -31,10 +36,15 @@ function setupPromises(scheduleMicrotasks) {
       const uid = promiseToGuidProperty.get(promise);
       promiseToGuidProperty.delete(promise);
       if (hasBeenNotified === true) {
+        let warning = null;
+        if (!process.listenerCount('rejectionHandled')) {
```

Comment:
@Fishrock123 Are you referring to the code in the block below this one? That’s not technically a listener, so it isn’t counted here. Also, at least a quick check in the REPL yields `process.listenerCount('rejectionHandled') === 0` by default.